data/migrations/20190828030942_alter-organization-table.js

```diff
@@ -0,0 +1,12 @@
+exports.up = function(knex) {
+  return knex.schema.table('organizations', tbl => {
+    tbl.text('channel_name').notNullable();
+    tbl.text('channel_id').notNullable();
+    tbl.text('access_token').notNullable();
+    tbl.timestamp('installation_date').defaultTo(knex.fn.now());
+  });
+};
+
+exports.down = function(knex) {
+  return knex.schema.dropTableIfExists('organizations');
```

Comment:
Migrations are to make sure everyone has same db schema
migrate down should UNDO the changes (edited) 
just those changes in that migration file
so, don't just dropTableI. Undo the changes in this migration file.

Reply:
resolved